src/main/java/mekanism/common/CommonProxy.java

```diff
@@ -311,8 +311,10 @@ public void loadConfiguration()
 		general.OreDictPlatinum = Mekanism.configuration.get(Configuration.CATEGORY_GENERAL, "OreDictPlatinum", false).getBoolean();
 		general.enableSiliconCompat = Mekanism.configuration.get(Configuration.CATEGORY_GENERAL, "EnableSiliconCompat", true).getBoolean();
 
+		general.IC2SinkTier = Mekanism.configuration.get(Configuration.CATEGORY_GENERAL, "IC2SinkTier", 4, "Tier level for machines, we recommend setting it to tier 2 for ic2 classic", 1, 4).getInt();
+		general.IC2SourceTier = Mekanism.configuration.get(Configuration.CATEGORY_GENERAL, "IC2SourceTier", 4, "Tier level for generators, we recommend setting it to tier 2 for ic2 classic", 1, 4).getInt();
 
-		String s = Mekanism.configuration.get(Configuration.CATEGORY_GENERAL, "EnergyType", "J", null, new String[]{"J", "RF", "MJ", "EU"}).getString();
```

Comment:
Why is Joules getting removed in an IC2 oriented update?

Reply:
default is changed to rf, i forgot that i was on this branch when committing.

Reply:
Is the config still parsing power related entries as Joules?

Reply:
yes, i just changed the default setting to rf from j, all of them still works. changed it due to the large amount of modpacks using rf and changing to rf will bring some cohesion in the experience.